### ISSUE TITLE
The light color of the new modules isn't saved

### ISSUE THREAD

**Issue by ERYK500** [Panakotta00/FicsIt-Networks#167]
**Describe the bug**
The color (set via [Module trace here]:setColor(R,G,B,E)) isn't saved (or at least isn't re-loaded on a save load).
This affects the following modules:
- MicroDisplay;
- Indicator;
- 3PosSwitch (potentialy also the 2Pos variant).

**To Reproduce**
1. Build this computer and a module panel with any module/s mentioned above
2. Connect them together
3. Set the PC's EEPROM to:
```
local temp = component.proxy(--[[the uuid of the panel goes here]]):getModules()
for _,v in ipairs(temp) do
    v:setColor(1,0,0,1) -- set the color to any other than the default
end
```
5. Run the code.
6. Save the session
7. (Optional) Quit and restart Satisfactory
8. Reload the game from the newly made save
9. Observe the modules

**Expected behavior**
The color should persist between game sessions.

**Comment by RozeDoyanawa** [comment]
This was by design. I'm working on changing it.

**Comment by RozeDoyanawa** [comment]
Probably fixed :3

**Comment by RozeDoyanawa** [comment]
Fixed in #168
